### PR TITLE
Updated Appsflyer Device Mode related information

### PR DESCRIPTION
**Pull request by desusai7**
## Updated Appsflyer Device Mode related information

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/attribution/appsflyer.mdx [hunk 1]
```diff
@@ -317,7 +330,7 @@ var onDeepLinkCanceller = onDeepLink((data) => {
 })
 ```
 </span>
-As seen above, each of the listeners returns a function that - whenever called - removes those listeners.
+As seen above, each of the listeners returns a function that removes those listeners whenever it is called.
```
Comment (ameypv-rudder) [inline destinations/attribution/appsflyer.mdx:333]:
```suggestion

```

Comment (ameypv-rudder) [reply destinations/attribution/appsflyer.mdx:333]: @desusai7 can we remove this sentence? It's quite confusing, currently.

**Review by ameypv-rudder — APPROVED** [review]

File: destinations/attribution/appsflyer.mdx (hunk 1 above)
Comment (desusai7) [reply destinations/attribution/appsflyer.mdx:333]: Okay Sure

File: destinations/attribution/appsflyer.mdx [hunk 2]
```diff
@@ -167,7 +180,20 @@ implementation 'com.android.installreferrer:installreferrer:2.+'
 ```
 </span>
 </li>
-<li>Finally, change the initialization of your <code class="inline-code">RudderClient</code> in your <code class="inline-code">Application</code> class, as shown:
+<li>Starting from AppsFlyer iOS device mode version <code class="inline-code">2.0.0</code> and above</strong>, the RudderStack SDK <strong>does not</strong> automatically initialize the AppsFlyer SDK. You need to initialize the AppsFlyer SDK as shown:
```
Comment (desusai7) [inline destinations/attribution/appsflyer.mdx:183]:
```suggestion
<li>Starting from AppsFlyer Android device mode version <code class="inline-code">2.0.0</code> and above</strong>, the RudderStack SDK <strong>does not</strong> automatically initialize the AppsFlyer SDK. You need to initialize the AppsFlyer SDK as shown:
```

File: destinations/attribution/appsflyer.mdx [hunk 3]
```diff
@@ -145,7 +158,7 @@ RSConfigBuilder *builder = [[RSConfigBuilder alloc] init];
       <TabPanel>
         To add AppsFlyer to your Android project:
 <ol>
-<li>Add the repository as shown:
+<li>Add the <code class="inline-code">mavnCentral()</code> repository, as shown:
```
Comment (ishachopra0401) [inline destinations/attribution/appsflyer.mdx:161]:
```suggestion
<li>Add the <code class="inline-code">mavenCentral()</code> repository, as shown:
```

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**